### PR TITLE
Articles page

### PR DESCRIPTION
**Pull request by SuperViolinsADS**

### CODE REVIEW

File: public/rules.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" />
-  <link rel="stylesheet" href="rules.css" />
+  <link rel="stylesheet" href="articles.css" />
```
Comment (SuperViolinsADS) [inline public/rules.html:7]: Wrong file. Change this back and change the stylesheet in articles.html

File: public/articles.css [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  background-color: var(--color-black);
+}
+
+.hero {
+  height: calc(100vh - 80px);
+  background-image: url(./websiteBG.png);
+  background-position: center;
+}
+
+.ruletitle {
```
Comment (SuperViolinsADS) [inline public/articles.css:115]: Add some styles for ".articletitle" similarly to this

File: public/articles.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" />
+  <link rel="stylesheet" href="rules.css" />
```
Comment (SuperViolinsADS) [inline public/articles.html:7]: Change to articles.css

File: public/articles.css (hunk 2 above)
Comment (SavinienBrassheart) [reply public/articles.css:115]: It'd be nice if there was a button that just let me apply to proposed changes

**Review by PokeCat420 — APPROVED** [review]
👍

**Pull request merged**